JSQMessagesViewController/Views/JSQMessagesComposerTextView.m

```diff
@@ -163,6 +164,11 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
 - (void)paste:(id)sender
 {
     if (!self.pasteDelegate || [self.pasteDelegate composerTextView:self shouldPasteWithSender:sender]) {
+        if (!shouldPasteRTF) {
+            [self setText:[UIPasteboard generalPasteboard].string];
```

Comment:
When it is a photo... what does the .string give us?

Reply:
nil